lib/util/naming.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Nice! Good refactoring. However, I think `config/config-file.js` can also be updated to use this.

Reply:
1. Did you intend for this to happen within this current pull request?
2. Just to clarify the scope of your request, this is within `normalizePackageName`?

Reply:
Ok I moved the normalizePackageName to the naming util altogether. It simplified the logic in getFormatter quite a bit.